source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -80,7 +102,34 @@ Extractor::extract(Http::StreamFilterCallbacks &callbacks,
     if (header_entries.empty()) {
       return "";
     }
-    return extractValue(callbacks, header_entries[0]->value().getStringView());
+    const auto &header_value = header_entries[0]->value().getStringView();
```

Comment:
What is the purpose of pulling this out into a `const auto&` instead of inlining? I see that is done for the invocations of the newly added methods too

Reply:
I think this is left over from me trying to work around the issue of returning an `absl::string_view` while using the new destructive extraction methods. I'll switch it back to where it was before

Reply:
Anyway, updated that here: https://github.com/solo-io/envoy-gloo/pull/309/commits/5fcb54524240eac9fb9f56b609e1aa0d829513f4#diff-8c1e447dd62e766fbdd2f928d80d5332bdc2defbfd353a41f35db35fcd3b7deaR105